project/Dependencies.scala

```diff
@@ -0,0 +1,48 @@
+import sbt._
+
+object Dependencies {
+  val akkaV = "2.3.6"
+  val sprayV = "1.3.2"
+  val slickV = "3.1.1"
+
+  val modelDependencies = Seq(
+    "io.spray" %% "spray-json" % "1.3.1",
+    "io.spray" %% "spray-http" % sprayV,
+    "joda-time" % "joda-time" % "2.9.4",
+    "org.joda" % "joda-convert" % "1.8",
+    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
+    "org.scalatest" %% "scalatest" % "2.2.4" % "test"
+  )
+
+  val rawlsDependencies = modelDependencies ++ Seq(
+    "com.typesafe" % "config" % "1.3.0",
+    ("com.gettyimages" %% "spray-swagger" % "0.5.0").exclude("com.typesafe.scala-logging", "scala-logging-slf4j_2.11").exclude("com.typesafe.scala-logging", "scala-logging-api_2.11").exclude("com.google.guava", "guava"),
```

Comment:
Perhaps convert to the more conventional infix DSL:

```scala
"com.gettyimages" %% "spray-swagger" % "0.5.0"
  exclude("com.typesafe.scala-logging", "scala-logging-slf4j_2.11")
  exclude("com.typesafe.scala-logging", "scala-logging-api_2.11")
  exclude("com.google.guava", "guava"),
```

Same goes for all other calls to the `%` operator that end up wrapped in parenthesis below.